plugins/callback/profile_tasks.py

```diff
@@ -199,3 +199,4 @@ def playbook_on_stats(self, stats):
             if 'path' in result:
                 msg += u"\n{0:-<{1}}".format(result['path'] + u' ', self._display.columns)
             self._display.display(msg)
+            
```

Comment:
To pass the sanity test, you need to remove the whitespace at the end of the line.